### PR TITLE
blog post announcing pyladies member registration

### PR DESCRIPTION
**Pull request by lorenanicole**
# PyLadies Pull Request
_Please document the following. We will aim to be as timely as possible in processing your pull request, however as this is a volunteer staffed project anticipate at least a week for pull requests to start to be reviewed._ 

## Is this pull requesting adding or updating a [PyLadies chapter information](https://github.com/pyladies/pyladies#to-add-a-new-pyladies-location)? 

- [ ] Yes
- [X] No

### If you marked yes, is this chapter registered in the PyLadies Chapter Directory?

- [ ] Yes
- [ ] No

### If you marked no, please register your chapter using the following form [here](https://forms.gle/LYT6TzjgJ437Y4mr8). The form can only be accessed by being logged into your official PyLadies email account. Were you able to complete the registration?

- [ ] Yes
- [ ] No, I could not because [add response below]:

## Have you confirmed the website builds locally with your chapters? See [here](https://github.com/pyladies/pyladies#run-the-site-locally).

- [X] Yes
- [ ] No

## Does your pull request pass the `ci/circleci: build` check? See [this example pull request, in the `All checks have passed` section](https://github.com/pyladies/pyladies/pull/461).

- [X] Yes
- [ ] No

# Pull Request Checklist

- [X] Fill out all sections above
- [x] Share pull request in PyLadies Slack channel #project-tech-infra

/cc @pyladies/tech-and-infra-admins

### CODE REVIEW

File: www/_posts/2020/Announcement for registeringpyladies-members.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ 
+As  of today we have registration available in:
+
+- [English](https://forms.gle/f3M4JUzA7JH48Swo8)
+- [Spanish / Español](https://forms.gle/CaDhPsjLgEmrqV7RA)
+- Portuguese / Português](https://forms.gle/9AdTdBr67ikiAFXSA) 
+
+We’re currently in the process of translating our registration documentation into Spanish, Portuguese, Mandarin (Chinese), Arabic, and French. If you’d like to help with translation, let us know by adding your voice to the GitHub issue [here](https://github.com/pyladies/global-organizing/issues/54).
+
+
```
Comment (Mariatta) [inline www/_posts/2020/Announcement for registeringpyladies-members.md]: Add paragraph to thank translators: 
- Portuguese: Débora Azevedo
- Spanish: Diana Rodriguez
- Russian: Volodymyr Kirichinets
- Chinese: Georgi Ker
- French: Georgi Ker

And thanks to the interim global team :)

File: www/_posts/2020/Announcement for registeringpyladies-members.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+As we shared on our blog in [August 2019]( https://www.pyladies.com/blog/Important-PyLadies-Update/important-pyladies-update/), PyLadies  has been hard at work, revamping our PyLadies presence as well as our global leadership. Why? To enable us to better run our chapters ,   not only locally but globally. 
+The intent is to bring together all of our local chapters, to help us collaborate, to share ideas and also start a [PyLadies GlobalCouncil]( https://www.pyladies.com/blog/Help-us-decide-our-PyLadies-Projects-and-PyLadies-Global-Council-Selection-Process/help-us-with-pyladies-project-and-council-selection/) , which will be responsible for overseeing the Global PyLadies Community.
+
+To do this, however, we need your help! We want to engage with our PyLadies members!  Starting today, PyLadies members can register to become a basic member or a voting member.
+ 
+As  of today we have registration available in:
+
+- [English](https://forms.gle/f3M4JUzA7JH48Swo8)
+- [Spanish / Español](https://forms.gle/CaDhPsjLgEmrqV7RA)
+- Portuguese / Português](https://forms.gle/9AdTdBr67ikiAFXSA)
```
[the remark this answers is not in the document]
Comment (lorenanicole) [reply www/_posts/2020/Announcement for registeringpyladies-members.md]: @BethanyG did we make a form for this? I'm not sure I've  seen it yet.

**Comment by Mariatta** [comment]
Some characters aren't being rendered correctly ☹️  Does mynt not support UTF-8?
Screen Shot 2020-03-05 at 11 15 21 AM

**Comment by Mariatta** [comment]
Please ensure to add `Co-Authored-By: Abigail Dogbe
 <mesrenyame12@gmail.com>` when you merge the PR @lorenanicole

**Pull request merged**